tests/test_definitions/multicolumn_map_expectations/expect_compound_columns_to_be_unique.json

```diff
@@ -213,7 +213,7 @@
     },{
       "title": "unexpected_values_exact_match_out_without_unexpected_index_list",
       "exact_match_out" : true,
-      "suppress_test_for": ["pandas"],
+      "only_for": ["sqlalchemy"],
```

Comment:
`"exact_match_out" : true` will no longer work for spark since we are no longer sorting results